test/run_all_tests.sh

```diff
@@ -114,14 +114,21 @@ else
   all_tests_result=999
 fi
 
-echo
```

Comment:
I originally added this blank line made it easier to read the tests results, and it matches the documentation. However, I don't mind to remove it if you feel strongly about it.